ui/Screen/Project.svelte

```diff
@@ -31,11 +31,44 @@
 
   export let urn: Urn;
   export let activeView: router.ProjectView = { type: "files" };
+  let hoverstyle: string = "";
+
+  const mouseenter = () => {
+    hoverstyle = "background-color: var(--color-foreground-level-2)";
```

Comment:
Ingenious! I wonder if there's a way to put this logic into the button itself?

Reply:
I mean I'm only doing it here cause I'm overwriting the button hover style cause the background is the same color as the hover background color.

Reply:
Wonder if we'll also need to override it in other places later? Not a biggie tho.